packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/apifetures/Fetures.kt

```diff
@@ -0,0 +1,6 @@
+package expo.modules.kotlin.apifetures
```

Comment:
Just a nitpick, but I assume you meant to use "features" instead of "fetures"? Disregard if this is something I'm not aware of haha